### PR TITLE
Fork: no get gas cost, increase cost per send

### PR DESCRIPTION
**Pull request by Kubuxu**

### CODE REVIEW

File: build/forks.go [hunk 1]
```diff
@@ -7,3 +7,5 @@ const ForkFrigidHeight = 7950
 const ForkBootyBayHeight = 11000
 
 const ForkMissingSnowballs = 34000
+
+const ForkGoGreen = 36000
```
Comment (Kubuxu) [inline build/forks.go]: Needs to be updated

**Comment by whyrusleeping** [comment]
@Kubuxu I think we still want to charge gas for reads in general, but for specific system calls (like looking up ID addresses) we want to just charge a fixed cost (so we can cache those).

**Comment by whyrusleeping** [comment]
Basically, I want the stateTree.LookupID method to have a fixed gas cost, and a cache strapped on top, but the rest of the reads in the system continue along the same way we're doing now.

**Comment by Kubuxu** [comment]
Hmmm, looking at the code, this isn't needed. There is no get gas cost for `state.LookupID` because stateTree does not use gas charging datastore.

**Pull request closed without merging**